lib/glimesh/chat.ex

```diff
@@ -169,6 +169,17 @@ defmodule Glimesh.Chat do
     |> Enum.reverse()
   end
 
+  def list_chat_messages(channel, limit) do
```

Comment:
Should we combine this into the function above and add a default limit of 5?

Reply:
I was thinking of doing that but ended up going with this. Either way works for me.